lib/compat/wordpress-6.7/rest-api.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * PHP and WordPress configuration compatibility functions for the Gutenberg
+ * editor plugin changes related to REST API.
+ *
+ * @package gutenberg
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	die( 'Silence is golden.' );
+}
+
+if ( ! function_exists( 'wp_api_template_registry' ) ) {
```

Comment:
How it would be possible for this function to already exist?

Reply:
Good catch. Removed it in f7f65a027a9ea55d54900654c46767a5df719381.

Reply:
Well, actually PHP Coding Standards complain if we don't guard it against redeclaration. :thinking: 

I revert it for now in 2640ff1cdd1823ac6b076d25bdff299ad95f7cf3.